CHANGELOG.md

```diff
@@ -4,9 +4,20 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.40.0]
+
+### What's Changed
+
+* Ease setup in Amazon Linux 2 by @adpaco-aws in https://github.com/model-checking/kani/pull/2833
+* Propogate backend options into goto-synthesizer by @qinheping in https://github.com/model-checking/kani/pull/2643
+* Update CBMC version to 5.95.1 by @adpaco-aws in https://github.com/model-checking/kani/pull/2844
+* Fix issues and Update toolchain 10-26 by @jaisnan in https://github.com/model-checking/kani/pull/2843
```

Comment:
Didn't we agree on using the format:
```
Rust toolchain upgraded to `nightly-20yy-mm-dd` by X, Y, and Z
```
?

Reply:
Yes, but only if #2846 made it, otherwise we'd have had a single contributor. Since #2846 was merged, I've changed it to use that format.